### PR TITLE
Improve explanations on available releases

### PR DESCRIPTION
**Pull request by farshidtz**
[PR Preview](https://canonical-matter--31.com.readthedocs.build/en/31/)

### CODE REVIEW

File: tutorial/pi-gpio-commander.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ SSH to the Raspberry Pi and install the snap:
 sudo snap install matter-pi-gpio-commander
 ```
 
+We can see the available releases on the store page or with: ``snap info matter-pi-gpio-commander``.
+The releases are accessible via the different [channels](https://snapcraft.io/docs/channels).
+
```
Comment (jpm-canonical) [inline tutorial/pi-gpio-commander.md:28]: This whole paragraph feels a little out of place here. It interrupts the flow of the guide. Maybe simplify it to just a note, as the document we link to already describes what channels are and how to list available versions.

> Pre-release versions of matter-pi-gpio-commander is available in different [channels](https://snapcraft.io/docs/channels).

File: how-to/chip-tool-commission-and-control.rst [hunk 2]
```diff
@@ -26,8 +26,8 @@ Install the Chip Tool snap:
 
    sudo snap install chip-tool
 
-You can choose another `channel <https://snapcraft.io/docs/channels>`_
-to install a development release.
+You can choose another `channel <https://snapcraft.io/docs/channels>`_ to install a development release.
+Check the available releases from the store page or with: ``snap info chip-tool``
```
Comment (jpm-canonical) [inline how-to/chip-tool-commission-and-control.rst:30]: The docs on snapcraft channels already describe how to list available releases and how to install them. I'm not sure adding the list command here makes it clearer, as the second step of installing it is still missing. Maybe better to leave the paragraph as is, or omitting it completely.

**Review by jpm-canonical — CHANGES_REQUESTED** [review]
In my opinion these changes do not make the documentation more user friendly. In fact we are only providing half of the info (list available versions), while the other half of the info (installing another version) needs to be found in the linked documentation.

I would say we remove these paragraphs about other versions completely to make it easier for a new user to understand. Then we can add a new page or paragraph at the end about pre-release versions, discussing: 
* what they are
* how to interpret their version numbers 
* how to install them

That way power users can find the info, but we don't complicate the main flow of the docs for newcomers.

**Comment by farshidtz** [comment]
I agree with the comments. Could add a new explanation doc in the future, but I don't see that necessary right now.

**Review by jpm-canonical — APPROVED** [review]
Yes this is nicer for now

**Pull request merged**